test/functional/unified-spec-runner/entities.ts

```diff
@@ -329,7 +330,10 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
         const useMultipleMongoses =
           (config.topologyType === 'LoadBalanced' || config.topologyType === 'Sharded') &&
           entity.client.useMultipleMongoses;
-        const uri = config.url({ useMultipleMongoses });
+        const uri = makeConnectionString(
```

Comment:
Moving this out of the constructor assisted with debugging, there was a bit too much inlined before